packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -77,23 +86,44 @@ function MainLayoutHeader({
     campaignKey: ReferralCampaignKey.Search,
   });
 
+  const { currentStreak } = useReadingStreak();
+
   const renderButtons = () => {
     return (
       <>
-        <CreatePostButton
-          className={classNames(
-            'mr-0 laptop:mr-3',
-            optOutWeeklyGoal ? 'tablet:mr-0' : 'tablet:mr-3',
-          )}
-        />
+        <CreatePostButton />
+        {user && readingStreak === ReadingStreaksExperiment.V1 && (
+          <SimpleTooltip
```

Comment:
I would go with adding this logic to a separate component called `ReadingStreakButton`, similar to `CompanionPopupButton`, but up to you in the end

Reply:
Agreed. This is to avoid cluttering the main component.